app/managers.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+from typing import List
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(
+            self,
+            first_name: str,
+            last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES(?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> List[Actor]:
+        actors_data = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"

```

Comment:
It's better to follow SQL style guidelines even while writing raw SQL in Python. Therefore I'd suggest to start FROM clause from new line.